test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -501,17 +485,25 @@ def set_vntype(self, typestring='myvntype'):
         """
         pbs_home = self.server.pbs_conf['PBS_HOME']
         vntype_file = os.path.join(pbs_home, 'mom_priv', 'vntype')
+        self.logger.info("Setting vntype to %s in %s" %
+                         (typestring, vntype_file))
+        cmd = ['/bin/echo', '%s' % typestring]
+        self.logger.info("Command is: %s" % (cmd))
```

Comment:
This is unneeded.  It is logged at INFOCLI which is on by default.

Reply:
Removed.